tools/script_template.py

```diff
@@ -44,6 +44,16 @@ def parse_flux_lines(flux_lines):
     flux_array = all_entries.reshape(num_intervals, num_groups)
     return flux_array
 
+def calc_flux_mag_flattened(total_flux, active_burn_time, t_irr_arr):
+    '''
+    Calculates an average flux magnitude for each interval defined in ALARA.
+    The flux is averaged over the total amount of time elapsed between the start of the first pulse and the end of the last.
+    inputs:
+        total_flux : value of the flux summed over all energy bins''
```

Comment:
Can you add a comment about the returned variable and its shape? It's not obvious to me...